statsmodels/genmod/families/family.py

```diff
@@ -798,42 +806,35 @@ def deviance(self, endog, mu, scale=1.):
             Fitted mean response variable
         scale : float, optional
             An optional scale argument
-
         Returns
         --------
         deviance : float
             The deviance function as defined below
-
         Notes
         -----
         If the endogenous variable is binary:
-
         `deviance` = -2*sum(I_one * log(mu) + (I_zero)*log(1-mu))
-
```

Comment:
Oh shoot. Not sure why that happened. I presume I can just copy-paste it back from the original. The math was the hard part. Writing code is easy.

Is that a setting in spyder? That's what I've historically used. It's buggy at times but is really easy to use.